pensieve/analysis.py

```diff
@@ -95,14 +95,23 @@ def _get_timelimits_if_ready(
         ):
             return None
 
+        analysis_length_dates = (
+            (self.config.experiment.end_date - self.config.experiment.start_date).days
+            - dates_enrollment
+            + 1
+        )
+
+        if analysis_length_dates < 0:
+            logging.error(
+                "Proposed enrollment longer than analysis dates length:"
+                + f"{self.config.experiment.experimenter_experiment.slug}"
```

Comment:
```suggestion
                + f"{self.config.experiment.slug}"
```

I should really make those members of ExperimentConfig private.